libraries/helpers.rb

```diff
@@ -190,6 +190,9 @@ def validate_os_type
   when 'centos'
     current_os = "centos#{node['platform_version'].to_i}"
     raise_os_not_match(current_os, node['cluster']['base_os']) if node['cluster']['base_os'] != current_os
+  when 'redhat'
```

Comment:
out-of-scope for this PR but we need to move this code to a resource as it is OS-dependent.

Reply:
Agreed, adding to the task.